### ISSUE TITLE
stamppFst function fails with example data in R v4.3.0

### ISSUE THREAD

**Issue by monis4567** [lpembleton/StAMPP#3]
I want to make use of the stamppFst. I tried it out on 'Ubuntu 22.04.2 LTS', using R version 4.3.0 (2023-04-21) -- "Already Tomorrow", on a Platform: x86_64-pc-linux-gnu (64-bit).

This below  is from my terminal:
________________
> library("StAMPP")
Loading required package: pegas
Loading required package: ape

Attaching package: ‘pegas’

The following object is masked from ‘package:ape’:

    mst

Registered S3 method overwritten by 'ade4':
  method      from 
  print.amova pegas
> data(potato.mini, package="StAMPP")
> potato.freq <- stamppConvert(potato.mini, "r")
> potato.fst <- stamppFst(potato.freq, 100, 95, 1)
Rscript execution error: No such file or directory
^C
> quit()
Save workspace image? [y/n/c]: n
_________________
Any suggestions to how this can be made working? Thanks

**Comment by lpembleton** [comment]
That is strange. I have set up a virtual machine with your exact system and R version, however, the StAMPP commands run with no error. I suspect something is broken with your R installation. Did you follow the guide on CRAN (https://cran.r-project.org/bin/linux/ubuntu/) for installing R in Ubuntu? If not, maybe remove R and try installing it again. 
Hopefully that works 🤞

**Comment by monis4567** [comment]
Hi, Thanks for the fast reply.

It might be my version of R. I have had some issues over the past months after I upgraded to R v4.3.
I will try re-installing R, and let you know if I can get it working.

Thanks
Steen
________________________________
Fra: Luke Pembleton ***@***.***>
Sendt: 22. juni 2023 06:53
Til: lpembleton/StAMPP ***@***.***>
Cc: Steen Knudsen ***@***.***>; Author ***@***.***>
Emne: Re: [lpembleton/StAMPP] stamppFst function fails with example data in R v4.3.0 (Issue #3)


CAUTION:EXTERNAL EMAIL - Do not click links or attachments unless you recognize the sender


That is strange. I have set up a virtual machine with your exact system and R version, however, the StAMPP commands run with no error. I suspect something is broken with your R installation. Did you follow the guide on CRAN (https://cran.r-project.org/bin/linux/ubuntu/) for installing R in Ubuntu? If not, maybe remove R and try installing it again.
Hopefully that works 🤞

**Comment by monis4567** [comment]
I discovered that I was getting an extra unwanted PATH added to my $PATH that bash is using.

On stackoverflow I found a suggestion, to find out which file was adding this unneeded extra path
With this command in a terminal 

` $ grep --color -H 'PATH=' ~/.bashrc ~/.profile ~/.bash_profile ~/bash.login                      ~/.bash_aliases /etc/bash.bashrc /etc/profile                      /etc/profile.d/* /etc/environment 2> /dev/null
`
I discovered the problem was in '/etc/profile.d/rstudio.sh'
So I edited this file using
 $ sudo nano /etc/profile.d/rstudio.sh

and just deleted all paths here, so they are not added to my $PATH, and this allowed Rstudio to find the correct Rscript where it is placed, instead  of looking for it in some odd temporary directory that Rstudio once installed. This sorted this issue for my entire Rstudio setup, and for issues I had with installing other packages that also were dependent on Rscript.
